extensions/sql-migration/src/dashboard/dashboardTab.ts

```diff
@@ -143,6 +143,7 @@ export class DashboardTab extends TabBase<DashboardTab> {
 		toolbar.addToolbarItems([
 			<azdata.ToolbarComponent>{ component: this.createNewMigrationButton() },
 			<azdata.ToolbarComponent>{ component: this.createNewLoginMigrationButton() },
+			<azdata.ToolbarComponent>{ component: this.createImportMigrationButton() },
```

Comment:
if this is the first PR of several for this feature, should we disable / comment out the button for now until it's ready? Or did you plan on iterating on this PR and not checking in until it's ready?

Reply:
Correct, I plan to iterate on this PR and check in until its ready. Will remove draft when ready then.